recipes/find-kedro/meta.yml

```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "find-kedro" %}
+{% set version = "0.0.3" %}
```

Comment:
it looks like the most recent release is `0.0.4`

Reply:
I updated to 0.0.5